server/static/css/style.css

```
@@ -88,8 +88,1468 @@ button:disabled,
+  --color-text-primary: var(--admin-text);
+  --color-text-secondary: var(--admin-text-dim);
+  --color-text-muted: var(--admin-text-dim);
+  --color-text-bright: var(--admin-text);
+  --color-text-strong: var(--admin-text);
+  --color-bg-deep: var(--admin-raised);
+  --color-bg-elevated: var(--admin-raised);
+  --color-bg-base: var(--admin-panel);
+  --hud-line: var(--admin-line);
+  --hud-line-strong: rgba(15, 23, 42, 0.18);
+  --hud-cyan-soft: rgba(56, 189, 248, 0.12);
+  color-scheme: light;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)